packages/ui/src/pages/QueuePage/QueuePage.tsx

```diff
@@ -47,6 +50,7 @@ export const QueuePage = ({
         <JobCard
           key={job.id}
           job={job}
+          jobUrlPath={`/queue/${encodeURIComponent(queue.name)}/${job.id}${search}`}
```

Comment:
can do - this again conflicts with some old types deep in the app
`QueueJobJson["id"]` can be `null` for some reason. I can't think of how or why this would happen. -- I'll make the change but it's ugly -> 
```
encodeURIComponent(job.id ?? '')
```